frontend/dockerfile/cmd/dockerfile-frontend/hack/release

```diff
@@ -84,7 +91,7 @@ case $TYP in
     pushTag=${pushTag}-$TAG
   fi
 
-  buildxCmd build $platformFlag $cacheFromFlags $cacheToFlags $(buildAttestFlags) \
+  buildxCmd build $platformFlag $cacheFromFlags $cacheToFlags $nocacheFlag $(buildAttestFlags) \
```

Comment:
Hm... so we allow both `--cache-xxx` and `--no-cache` to be set at the same time? 😬 Is that documented as a valid combination? (really wondering if we should've allowed that, as that can be rather ambiguous)

Reply:
Oh, nevermind; I guess we're resetting the other ones 🤦 - sorry didn't look properly

Reply:
`--no-cache`takes precedence afaik but yes just setting `cacheFromFlags` / `cachetoFlags` to empty is ok too.

Reply:
If it's accepted, we should probably look at that (I can definitely see that as "confusing" / "unexpected" behaviour).